components/scream/cime_config/config_compsets.xml

```diff
@@ -33,4 +33,16 @@
       <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
+      <alias>F2010-SCREAMv1-noSPA</alias> <!-- Coupled land-atm compset (using SCREAMv1), 2010 initial conditions -->
+      <lname>2010_SCREAM_ELM%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV_SIAC_SESP%noSPA</lname>
+      <support_level>Experimental, under development</support_level>
+  </compset>
+
+  <compset>
+      <alias>F2000-SCREAMv1-AQP1-noSPA</alias> <!-- atm aquaplanet compset (using SCREAMv1), 2000 initial conditions -->
+      <lname>2000_SCREAM%AQUA_SLND_SICE_DOCN%AQP1_SROF_SGLC_SWAV%noSPA</lname>
```

Comment:
Similarly, add noSPA as part of the physics options, the lname would be

2000_SCREAM%AQUA-noAero_SLND_SICE_DOCN%AQP1_SROF_SGLC_SWAV